air/src/aux_table/bitwise_pow2/pow2.rs

```diff
@@ -0,0 +1,421 @@
+use crate::utils::{binary_not, is_binary};
+
+use super::{EvaluationFrame, FieldElement, POW2_TRACE_OFFSET};
+use core::ops::Range;
+use vm_core::{bitwise::POW2_POWERS_PER_ROW, utils::range as create_range};
+use winter_air::TransitionConstraintDegree;
+
+// CONSTANTS
+// ================================================================================================
+
+/// The number of constraints on the power of two co-processor.
+pub const NUM_CONSTRAINTS: usize = 24;
```

Comment:
Now that I'm seeing this: 24 constraints is a lot. Granted, these constraints are quite simple, but still for a single `pow2` operation feels like a bit of a waste.

Nothing to do about this now - but we should try to find a way to improve this in the future. Maybe we should even create an issue for this.

Reply:
Yeah, I think it's worth creating an issue to revisit this construction. I've created #200